src/SkautisExtension.php

```diff
@@ -29,29 +38,29 @@ public function loadConfiguration()
 	{
 		$container = $this->getContainerBuilder();
 		$config = $this->validateConfig($this->defaults);
-		$config['profiler'] = isset($config['profiler']) ? $config['profiler'] : !empty($container->parameters['debugMode']);
+		$config['profiler'] = $config['profiler'] ?? !empty($container->parameters['debugMode']);
 
 		$container->addDefinition($this->prefix('config'))
-			->setClass('Skautis\Config', array($config['applicationId'], $config['testMode'], $config['cache'], $config['compression']));
+			->setFactory(\Skautis\Config::class, array($config['applicationId'], $config['testMode'], $config['cache'], $config['compression']));
 
 		$container->addDefinition($this->prefix('webServiceFactory'))
-			->setClass('Skautis\Wsdl\WebServiceFactory');
+			->setType(WebServiceFactory::class);
 
 		$manager = $container->addDefinition($this->prefix('wsdlManager'))
-			->setClass('Skautis\Wsdl\WsdlManager');
+			->setType(WsdlManager::class);
 
 		$container->addDefinition($this->prefix('session'))
-			->setClass('Skautis\Nette\SessionAdapter');
+			->setType(SessionAdapter::class);
 
 		$container->addDefinition($this->prefix('user'))
-			->setClass('Skautis\User');
+			->setType(User::class);
 
 		$container->addDefinition($this->prefix('skautis'))
-			->setClass('Skautis\Skautis');
+			->setType(Skautis::class);
 
-		if ($config['profiler'] && (class_exists('Tracy\Debugger') || class_exists('Nette\Diagnostics\Debugger'))) {
+		if ($config['profiler'] && (class_exists(Debugger::class) || class_exists('Nette\Diagnostics\Debugger'))) {
```

Comment:
`Nette\Diagnostics\Debugger` je možné odstranit - bylo to tam kvůli kompatibilitě s nějakýma archaickýma verzema.